scripts/patch_metal_spec.py

```diff
@@ -168,7 +172,8 @@ def loadYaml(fn):
             del fixedSpec['components']['schemas'][s]['properties'][p]['default']
 
 # FIX 3. fix type and format for integers
-fixNumInt(fixedSpec)
+# most likely not necessary in 05/2023
```

Comment:
How do we confirm whether this is necessary?  Once we confirm that, let's delete the patch instead of commenting it out.

Reply:
The way to confirm is to run `make generate` code with the patch commented. If openapi generator succeeds, we are good. That's enough if the patch fixes some generate error. 

If the patch fixes code logic, then we need to run a test using the generated code. I use the script in `examples/`. If it's something more complex, ideally we should have some integration tests here, but I usually just install the generated pip package locally and run ansible-collection integration tests.

Reply:
I'll remove the patch, this is verified to be fixed.